code_samples/sequelize/currency/app.js

```diff
@@ -102,10 +102,10 @@ client.on('message', async msg => {
 
 		return msg.channel.send('```' +
```

Comment:
This as a whole should be converted to `message.channel.send(stuffHere, { code: true })` (both here and in the guide page).